pkgs/development/libraries/science/math/arpack/default.nix

```diff
@@ -49,6 +49,26 @@ stdenv.mkDerivation rec {
     install_name_tool -change libblas.dylib ${blas}/lib/libblas.dylib $out/lib/libarpack.dylib
   '';
 
+  # disable stackprotector on aarch64-darwin for now
+  # https://github.com/NixOS/nixpkgs/issues/127608
+  #
+  # build error:
+  #
+  # Building Fortran object CMakeFiles/cmTC_2526c.dir/testFortranCompiler.f.o
+  #     /nix/store/dhmsmvic5303500m046y4c9p90s2mddp-gfortran-wrapper-11.2.0/bin/gfortran    -c /tmp/nix-build-arpack-3.8.0.drv-0/source/build/CMakeFiles/CMakeTmp/testFortranCompiler.f -o CMakeFiles/cmTC_2526c.dir/testFortranCompiler.f.o
+  #     f951: Warning: command-line option '-iframework /nix/store/3li2kbxnbw9vry7k8y9yn8kbbr3pphz8-apple-framework-CoreFoundation-11.0.0/Library/Frameworks' is valid for C/C++/ObjC/ObjC++ but not for Fortran
+  #     f951: Warning: command-line option '-iframework /nix/store/3li2kbxnbw9vry7k8y9yn8kbbr3pphz8-apple-framework-CoreFoundation-11.0.0/Library/Frameworks' is valid for C/C++/ObjC/ObjC++ but not for Fortran
+  #     /private/tmp/nix-build-arpack-3.8.0.drv-0/ccuy4kLL.s:21:15: error: index must be an integer in range [-256, 255].
+  #             ldr     x0, [x0, ___stack_chk_guard];momd
+  #                              ^
+  #     /private/tmp/nix-build-arpack-3.8.0.drv-0/ccuy4kLL.s:46:15: error: index must be an integer in range [-256, 255].
+  #             ldr     x0, [x0, ___stack_chk_guard];momd
+  #                              ^
+  #     make[1]: *** [CMakeFiles/cmTC_2526c.dir/build.make:78: CMakeFiles/cmTC_2526c.dir/testFortranCompiler.f.o] Error 1
+  #     make[1]: Leaving directory '/private/tmp/nix-build-arpack-3.8.0.drv-0/source/build/CMakeFiles/CMakeTmp'
+  #     make: *** [Makefile:127: cmTC_2526c/fast] Error 2
```

Comment:
Again, we can't put the output of every build failure we fix into the comments otherwise nixpkgs would be ten times the size it is.

Reply:
Sure, removed.